tests/test_colo_model_local.py

```diff
@@ -49,7 +49,7 @@ def test_macosx_warning(fileutils, coloutils):
         RuntimeWarning,
         match="CPU pinning is not supported on MacOSX. Ignoring pinning specification."
     ):
-        colo_model = coloutils.setup_test_colo(
```

Comment:
General questions on tests:  do we have a test that checks for non-generated experiments with more than one colocated model?   That was how the bug was first discovered.

Reply:
I think we do, but not for local launcher. I'll add one.

Reply:
Done